version_control/katalyst_by_codurance/css/templates/home.css

```diff
@@ -0,0 +1,36 @@
+{% import '../tools/_breakpoints.css' as utils %}
+
+.hero-banner {
+    padding: var(--space-2xl) var(--space-s);
+}
+
+
+.hero-banner img {
+    margin-top: var(--space-l);
+    width: 100%;
+}
+
+{% call utils.medium_large_and_extra_large() %}
+    .hero-banner {
+        display: flex;
+        gap: var(--space-m);
```

Comment:
I've increased the spacing by one. So in this case it would be the L spacing. Also, keep in mind we use flexible spacing and the design is not updated